telegram/__init__.py

```diff
@@ -92,7 +93,6 @@
 from .shippingquery import ShippingQuery
 from .webhookinfo import WebhookInfo
 from .gamehighscore import GameHighScore
-from .videonote import VideoNote
```

Comment:
That's on me... I moved it long ago to fix a circular import issue I had... not sure why it's showing up in this diff though...